### ISSUE TITLE
Updating the status when deleting a vaccination is working only once per case 

### ISSUE THREAD

**Issue by KernB** [hzi-braunschweig/SORMAS-Project#10275]
### Bug Description

The solution introduced with the ticket 
https://github.com/hzi-braunschweig/SORMAS-Project/issues/9943
is working only once per case . 

### Steps to Reproduce

**Add a valide vaccination to the case** 
**Delete the vaccination**
Pop up window: Select a reason for deletion and confirm the deletion.  
Next pop up window: "Vaccination Status Update" Select "YES"
**Enter an new valide vaccination**
**Delete the second vaccination**
Pop up window: Select a reason for deletion and confirm the deletion.  

Instead of the next pop up window this message will appear:
![image](https://user-images.githubusercontent.com/101708230/188401509-2716e8a1-3a50-45a7-bae2-aada160e9b42.png)

### System Details

* Device: Laptop
* SORMAS version: 1.75.0
* Android version/Browser: Chrome
* Server URL:  https://test1401.sormas-oegd.de
* User Role:    Admin

**Comment by MateStrysewske** [comment]
@KernB I can't reproduce this behaviour on my machine. Can you please attach logs with the exception message?

**Comment by MateStrysewske** [comment]
Ah nvm, this is a duplicate of #10193.
